tests/scripts/thread-cert/config.py

```diff
@@ -51,6 +51,7 @@
                                 0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff])
 
 ADDRESS_TYPE = Enum('ADDRESS_TYPE', ('LINK_LOCAL', 'GLOBAL', 'RLOC', 'ALOC', 'ML_EID'))
+RSSI = {'LINK_QULITY_0': -100, 'LINK_QULITY_1': -95, 'LINK_QULITY_2': -85, 'LINK_QULITY_3': -65}
```

Comment:
What's this for?

Reply:
Enumerating typical RSSI used for later test cases which require special link quality, e.g. 5.3.5